features/view-transitions.yml

```diff
@@ -5,6 +5,13 @@ caniuse: view-transitions
 group: view-transitions
 compat_features:
   - api.Document.startViewTransition
+  - api.PageRevealEvent
```

Comment:
The [caniuse](https://caniuse.com/view-transitions) mentioned lists Chrome 111 and Safari 18. 

I think this needs a compute_from `api.ViewTransition` to match, or at the very least, the new keys added in Chrome 128 don't need to be moving back the date.

Reply:
Thanks, fixed the first bit.

Re #2435, I'll leave this as-is for now, which'll make it easier to find these related keys if that issue is fixed. Otherwise they'd be hidden away in drafts or BCD.